pylint/checkers/base/basic_checker.py

```diff
@@ -657,13 +657,35 @@ def _check_misplaced_format_function(self, call_node: nodes.Call) -> None:
             ):
                 self.add_message("misplaced-format-function", node=call_node)
 
+    @staticmethod
+    def _is_sys_exit(node: nodes.Call) -> bool:
+        """Detect call to sys.exit."""
+        if not isinstance(node.func, nodes.Attribute) or isinstance(
+            node.parent, nodes.Lambda
+        ):
+            return False
+
+        inferred = utils.safe_infer(node.func)
+
+        return inferred.qname() == "sys.exit" if inferred else False
+
     @utils.only_required_for_messages(
-        "eval-used", "exec-used", "bad-reversed-sequence", "misplaced-format-function"
+        "eval-used",
+        "exec-used",
+        "bad-reversed-sequence",
+        "misplaced-format-function",
+        "unreachable",
     )
     def visit_call(self, node: nodes.Call) -> None:
-        """Visit a Call node -> check if this is not a disallowed builtin
-        call and check for * or ** use.
+        """Visit a Call node ->
+        1
```

Comment:
Can remove empty line between 1. and 2.

Reply:
This was actually done by the [pre-commit ci](https://github.com/PyCQA/pylint/pull/7520/commits/f085811b8a4a6250bd5b323b54eff1e49fa54652) and I'm not sure how to undo it without it doing it again

Reply:
You need a summary for the docstring at the beggining for pep257, the following text won't be modified.